### PR TITLE
updated addresses of AccountOnboard, DataOnChain, ERC20Example

### PR DESCRIPTION
**Pull request by smiller-coti**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,11 @@
   "private": true,
   "scripts": {
     "postinstall": "cd confidentiality-contracts && yarn && yarn build",
-    "onboard": "ts-node src/main.ts",
     "erc20": "ts-node src/main.ts erc20",
     "dataOnChain": "ts-node src/main.ts dataOnChain"
   },
   "devDependencies": {
-    "@coti-io/coti-sdk-typescript": "https://github.com/coti-io/coti-sdk-typescript.git",
+    "@coti-io/coti-sdk-typescript": "https://github.com/coti-io/coti-sdk-typescript.git#11a38cf",
```
Comment (gmesika-coti) [inline package.json]: why is this pointing to a specific commit?

File: README.md [hunk 2]
```diff
@@ -89,7 +79,17 @@ The following process will help you run the [**`erc20.ts`**](https://github.com/
...
+> Ensure your environment meets all the pre-requisites. Visit the [pre-requisites section of the readme](https://github.com/coti-io/coti-sdk-typescript-examples/blob/main/README.md).
+
+### ERC20
+
+The following process will help you run the [**`erc20.ts`**](https://github.com/coti-io/coti-sdk-typescript-examples/blob/main/src/examples/erc20.ts) example from the [**COTI Typescript SDK Examples**](https://github.com/coti-io/coti-sdk-typescript-examples) project. The script includes functions for transferring tokens, approving allowances, and handling confidential transactions. The script uses various utilities from the SDK to manage confidential accounts, decrypt values, and ensure correct balances and allowances during transactions. It will also:
+
+* Create a EOA (Externally Owned Account)
+* Validate minimum balance
+
+1.  Run `erc20.ts` script
```
Comment (gmesika-coti) [inline README.md:92]: you deleted the numbering

File: README.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ The following process will help you run the [**`erc20.ts`**](https://github.com/
     \
     It is normal to receive the exception `Error: Please use faucet to fund account` on the first run. This will be resolved once the account is funded.  
 
-5. Head to the faucet at [**https://faucet.coti.io**](https://faucet.coti.io) to get devnet funds. \
+2. Head to the faucet at [**https://faucet.coti.io**](https://faucet.coti.io) to get devnet funds. \
```
Comment (gmesika-coti) [inline README.md:115]: here also

File: README.md [hunk 4]
```diff
@@ -121,12 +121,22 @@ The following process will help you run the [**`erc20.ts`**](https://github.com/
    \
    `<username> faucet transferred 5 COTIv2 (devnet)` \
    &#x20;
-6.  Run `erc20.ts` script once more
+3.  Run `erc20.ts` script once more
```
Comment (gmesika-coti) [inline README.md:124]: here also

**Review by gmesika-coti — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (smiller-coti) [reply package.json]: This example uses the SDK and it relied on pending changes from my PR in the TypeScript SDK repo to work properly. I mentioned this to you on Slack.

**Review by gmesika-coti — APPROVED** [review]

**Pull request merged**